libraries/AP_GPS/AP_GPS_UBLOX.cpp

```diff
@@ -234,11 +233,21 @@ AP_GPS_UBLOX::_verify_rate(uint8_t msg_class, uint8_t msg_id, uint8_t rate) {
                 _cfg_needs_save = true;
             }
             break;
+        case MSG_PVT:
+            if(rate == RATE_PVT) {
+                _unconfigured_messages &= ~CONFIG_RATE_PVT;
+            } else {
+                _configure_message_rate(msg_class, msg_id, RATE_PVT);
+                _unconfigured_messages |= CONFIG_RATE_PVT;
+                _cfg_needs_save = true;
+            }
```

Comment:
Well, unless the GPS doesn't allow it, then it is a shortcoming of the driver to depend on it. In any case, I think my purposed approach would still allow those other GPS drivers to make deletion based on whatever they want.